<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Check2FA</class>
 <widget class="QWidget" name="Check2FA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartQuizGen - Reset Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Check2FA {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                               stop:0 #6FB4E8, stop:1 #4A90E2);
}

QLabel#titleLabel {
    color: white;
    font-size: 32px;
    font-weight: bold;
    font-family: Arial, sans-serif;
}

QLabel#subtitleLabel {
    color: rgba(255, 255, 255, 200);
    font-size: 18px;
    font-family: Arial, sans-serif;
    margin-bottom: 20px;
}

QLabel {
    color: #2C3E50;
    font-size: 14px;
    font-weight: bold;
    font-family: Arial, sans-serif;
}

QLineEdit {
    background-color: white;
    border: none;
    border-radius: 25px;
    padding: 12px 20px;
    font-size: 14px;
    color: #666;
    font-family: Arial, sans-serif;
}

QLineEdit:focus {
    border: 2px solid #4A90E2;
    outline: none;
}

QPushButton {
    background-color: #2C3E50;
    color: white;
    border: none;
    border-radius: 25px;
    padding: 15px 30px;
    font-size: 16px;
    font-weight: bold;
    font-family: Arial, sans-serif;
    min-height: 20px;
}

QPushButton:hover {
    background-color: #34495E;
}

QPushButton:pressed {
    background-color: #1A252F;
}

QPushButton#checkCodeButton {
    background-color: #5DADE2;
    min-width: 100px;
    padding: 10px 15px;
    font-size: 14px;
}

QPushButton#checkCodeButton:hover {
    background-color: #4A90E2;
}

QFrame#contentFrame {
    background-color: rgba(255, 255, 255, 20);
    border-radius: 20px;
    padding: 20px;
}</string>
  </property>

  <!-- Main Title -->
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>SmartQuizGen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>

  <!-- Subtitle -->
  <widget class="QLabel" name="subtitleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Your Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>

  <!-- Content Frame -->
  <widget class="QFrame" name="contentFrame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>340</width>
     <height>400</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>

   <!-- 2FA Code Section -->
   <widget class="QLabel" name="Code2FAText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter 2FA Code</string>
    </property>
   </widget>

   <widget class="QLineEdit" name="Code2FA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>180</width>
      <height>45</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your 2FA code</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>

   <widget class="QPushButton" name="Check2FAButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>100</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>

   <!-- New Password Section -->
   <widget class="QLabel" name="NewPasswordText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>New Password</string>
    </property>
   </widget>

   <widget class="QLineEdit" name="NewPassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>300</width>
      <height>45</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your new password</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>

   <!-- Confirm Password Section -->
   <widget class="QLabel" name="ConfirmPasswordText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>300</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
   </widget>

   <widget class="QLineEdit" name="ConfirmPassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>300</width>
      <height>45</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Confirm your new password</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>

   <!-- Change Password Button -->
   <widget class="QPushButton" name="changePassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
